test/resources/verify.go

```diff
@@ -137,3 +137,29 @@ func verifyNoKnativeEventings(clients *test.Clients) error {
 	}
 	return nil
 }
+
+func KnativeEventingVerifyDeployment(t *testing.T, clients *test.Clients, names test.ResourceNames,
+	expectedDeployments []string) {
+	dpList, err := clients.KubeClient.Kube.AppsV1().Deployments(names.Namespace).List(metav1.ListOptions{})
+	assertEqual(t, err, nil)
+	assertEqual(t, len(dpList.Items), len(expectedDeployments))
```

Comment:
Can we use the names of the resources in the equality check? It'd be nice to know what resource leaked from the test failure.